lib/sth.js

```diff
@@ -84,31 +90,34 @@ function exitGracefully(err, callback) {
  */
 function startup(callback) {
   if (isStarted) {
-    return process.nextTick(callback);
+    if (callback) {
+      return process.nextTick(callback);
+    }
+    return;
   }
 
   var version = sthHelper.getVersion();
   sthLogger.info(
-    'Starting up STH server version %s...',
-    version.version,
     {
-      operationType: sthConfig.OPERATION_TYPE.SERVER_START
-    }
+      op: sthConfig.OPERATION_TYPE.SERVER_START
```

Comment:
You are repeating these objects over and over again. Better declare them as variables at the top of the page to improve readability of the rest of the file.